packages/mdctl-api/request.js

```diff
@@ -92,13 +92,17 @@ class Request {
       privates.request = response.request
       privates.response = response
       privates.result = result
-
       return result
-
-
     } catch (e) {
-      privates.error = Fault.from(e)
-      return Promise.reject(privates.error)
+      privates.response = e.response
+      if (e.response && e.response.data) {
+        if (!stream) {
+          privates.error = Fault.from(e.response.data)
+          return Promise.reject(privates.error)
+        }
+        return e.response.data.pipe(stream)
```

Comment:
not sure about this line. likely, the client would already have handled the data from the stream. thoughts?